<!---->
<launch>

<param name="use_sim_time" value="true" />

<!-- apply the sonar plant filter to the assembled sonar_cloud-->
<node pkg="sonar_cloud" type="sonar_plant_filter" output="screen" name="get_plant_positions">
	<param name="cloud_in" value="/sonar_cloud"/>
	<param name="cloud_out" value="/plant_poses"/>

	<param name="radius" value="0.1" />
	<param name="neigbour_nr" value="5" />
	<param name="distance" value="0.05" />

</node>
</launch>
